style.css

```diff
@@ -296,116 +348,278 @@ a:active {
 }
 
-footer{
+form {
+  opacity: 1;
+  width: 100%;
+  padding: 0.1rem 0 0 0;
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 64px 24px;
-  gap: 58px;
-  position: relative;  
-  width: 319px;  
-  height: 1143px;  
-  left: 386px;  
+  gap: 24px;
 }
 
 .footer-content {
-  width: 327px;
-  height: 363px;
-  font-family: 'Inter';
-  font-style: normal;
+  opacity: 1;
+  width: 100%;
   font-weight: 700;
   font-size: 40px;
   line-height: 52px;
   text-align: center;
-  color: #172B4D;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
+  color: #172b4d;
 }
 
 .footer-input {
   display: flex;
   flex-direction: column;
   align-items: center;
-  padding: 0px;
   gap: 24px;
-  width: 327px;
-  height: 330px;
-  flex: none;
-  order: 1;
-  flex-grow: 0;
+  padding: 10px 15px;
+  width: 100%;
+  opacity: 1;
 }
 
 .input {
   width: 100%;
   height: 48px;
-  flex: none;
-  order: 0;
-  flex-grow: 0;
-}
-
-.footer-input {
-  position: absolute;
-  height: 18px;
-  left: 16px;
-  right: 16px;
-  top: calc(50% - 18px/2);
-  font-family: 'Inter';
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  display: flex;
-  align-items: center;
-  color: #6B778C;
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
 }
 
 .input-area {
-  width: 327px;
+  background: url("images/Icon.png") no-repeat;
+  width: 100%;
   height: 114px;
-  flex: none;
-  order: 2;
-  flex-grow: 0;
+  border: 2px solid #acb7c3;
+  border-radius: 8px;
+  padding: 10px 15px;
 }
 
 .footer-email {
-  position: absolute;
-  width: 375px;
-  height: 189px;
-  font-family: 'Inter';
-  font-style: normal;
+  width: 100%;
   font-weight: 400;
   font-size: 20px;
   line-height: 32px;
-  top: 66rem;
-  left: 0;
   text-align: center;
-  color: #42526E;
-}
\ No newline at end of file
+  color: #42526e;
+  position: relative;
+  top: -3.5em;
+}
+
+.footer-media-link {
+  max-width: 170.17px;
+  margin: auto;
+}
+
+@media (min-width: 768px) {
+  .first-page {
+    background: url("images/bground.png") no-repeat center center;
+    -webkit-background-size: cover;
+    -moz-background-size: cover;
+    -o-background-size: cover;
+    background-size: cover;
+    min-width: 1472.8013916015625px;
+    align-items: center;
+    box-sizing: border-box;
+    margin: auto;
+  }
+
+  main {
+    margin: auto;
+    min-width: 1440px;
+    align-items: center;
+    opacity: 1;
+    box-sizing: border-box;
+  }
+
+  .detail-nav {
+    visibility: visible;
+    align-items: center;
+    gap: 8px;
+    width: 100%;
+    margin: auto;
+    opacity: 1;
+  }
+
+  .img-btn {
+    visibility: hidden;
+    opacity: 1;
+    width: 327px;
+    height: 48px;
+  }
+
+  .menu,
+  .logo {
+    visibility: hidden;
+  }
+
+  .nav {
+    margin: 0 auto;
+    width: 100%;
+  }
+
+  .detail-nav ul {
+    margin: 0 auto;
+    position: fixed;
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    gap: 8px;
+    justify-content: center;
+  }
```

Comment:
I think your content of your portfolio is scrolling into your `navbar`. 
This is a screenshot of your webpage.
<img width="1586" alt="Screenshot 2022-11-12 at 13 53 04" src="https://user-images.githubusercontent.com/99246359/201476959-64da9cc8-4f4f-467e-b072-03b9c6f9db23.png">
Can we fix this?